filebeat/inputsource/tcp/server_test.go

```diff
@@ -69,76 +69,76 @@ func TestReceiveEventsAndMetadata(t *testing.T) {
 		{
 			name:             "NewLine",
 			cfg:              map[string]interface{}{},
-			splitFunc:        netcommon.SplitFunc([]byte("\n")),
+			splitFunc:        streaming.SplitFunc([]byte("\n")),
```

Comment:
Are these tests in the right package? Looks like they should be in the streaming package.

Reply:
Well, in this format it is testing the whole TCP server. But I can move the testing of the splitting functionality to `streaming`.

Reply:
I take it back. This tests the whole `splitFunc` and `connectionHandler` functionality of the TCP server. I would keep it here as is.